concent_api/utils/helpers.py

```diff
@@ -165,10 +164,21 @@ def calculate_maximum_download_time(size: int) -> int:
 def calculate_subtask_verification_time(report_computed_task: message.ReportComputedTask) -> int:
     assert isinstance(report_computed_task, message.ReportComputedTask)
 
+    mdt = maximum_download_time(
+        size=report_computed_task.size,
+    )
+    ttc_dt = datetime.datetime.utcfromtimestamp(
+        report_computed_task.task_to_compute.timestamp,
+    )
+    subtask_dt = datetime.datetime.utcfromtimestamp(
+        report_computed_task.task_to_compute.compute_task_def['deadline'],
+    )
+    subtask_timeout = subtask_dt - ttc_dt
+
     return int(
-        subtask_verification_time(
-            report_computed_task
-        ).total_seconds()
+        (4 * settings.CONCENT_MESSAGING_TIME) +
+        (3 * mdt.total_seconds()) +
+        (0.5 * subtask_timeout.total_seconds())
```

Comment:
Could you add a comment explaining that we're not using `subtask_verification_time()` from golem-messages only because it has hard-coded values and we can't make it use values from our settings? And that it's only a temporary solution. Adding a link to the issue in golem-messages would be a good idea too.